articles/integrations/aws.md

```diff
@@ -271,7 +271,7 @@ The result of calling the delegation endpoint will contain the AWS token in the
 }
 ```
 
-**NOTE:** The Auth0 client libraries simplify calling these endpoints. See our [GitHub repo](https://github.com/auth0/) for the latest SDKs. See an example for client-side JavaScript at: [Delegation Token Request](https://github.com/auth0/auth0.js#delegation-token-request).
+**NOTE:** The [Auth0 client libraries](/libraries) simplify the process of calling these endpoints. See an example for client-side JavaScript at: [Delegation Token Request](https://github.com/auth0/auth0.js#delegation-token-request).
```

Comment:
@mpaktiti Actually, I just remembered we're making big changes to delegation. Should we mark this as blocked and waiting?